docs/api/exceptions.rst

```diff
@@ -0,0 +1,39 @@
+
+.. xdata:: datatable.exceptions
+    :src: --
+
+    This namespace contains warnings and exceptions that ``datatable`` may
+    throw during runtime.
+
+    Warnings are issued when it is helpful to inform the user of some condition
+    in a program, that doesn't result in an exception and the program termination.
+    Along with the built-in Python `warnings <https://docs.python.org/3/library/warnings.html>`_
```

Comment:
The only builtin warning that we issue is `FutureWarning` about deprecated features.

Reply:
Yes, but what I mean here is that we either issue a warning that is built-in, so no need to add it to the docs, or we issue something specific to datatable, then we add it to the docs.